docs/usage/nuget.md

```diff
@@ -39,15 +39,15 @@ Alternative feeds can be specified either [in a `NuGet.config` file](https://doc
 }
 ```
 
-If this example we defined 3 NuGet feeds.
-Packages resolving will process feeds consequentially.
-It means that if package will be resolved in second feed renovate won't look in last one.
+In this example we defined 3 NuGet feeds.
+Package resolving processes feeds in the listed order, from top to bottom.
+This means that if a package is resolved in the second feed, Renovate won't check the third and final feed.
```

Comment:
```suggestion
In this example we defined 3 NuGet feeds.
The package resolving process searches these 3 feeds for package updates and merge the results.
```

How about something like this?
I don't know what happens after the `merge` strategy is done, does the bot open a PR then?
Maybe we should cover that???

Reply:
`merge` means join all resolved versions to a single dependency (removing duplicate versions)

Reply:
```suggestion
In this example we defined 3 NuGet feeds.
The package resolving process uses the `merge` strategy to handle the 3 feeds.
All feeds are checked for dependency updates, and duplicate updates are merged/joined together into a single dependency update.
```

Reply:
Sounds good

Reply:
Opened PR #7841 to get this into the docs.